executor/mpp_gather.go

```diff
@@ -186,58 +74,27 @@ func (e *MPPGather) constructMPPTasksImpl(ctx context.Context, p *planFragment)
 	return allTasks, nil
 }
 
-func (e *MPPGather) constructSinglePhysicalTable(ctx context.Context, tableID int64, ranges []*ranger.Range) ([]*mppTask, error) {
+func (e *MPPGather) constructSinglePhysicalTable(ctx context.Context, tableID int64, ranges []*ranger.Range) ([]*kv.MPPTask, error) {
 	kvRanges := distsql.TableRangesToKVRanges(tableID, ranges, nil)
 	req := &kv.MPPBuildTasksRequest{KeyRanges: kvRanges}
-	stores, err := e.ctx.GetMPPClient().ConstructMPPTasks(ctx, req)
+	metas, err := e.ctx.GetMPPClient().ConstructMPPTasks(ctx, req)
```

Comment:
who 's meta?

Reply:
ConstructMPPTasks return a list of Tasks meta. That's natural.

Reply:
why `ConstructMPPTasks ` return metas, instead of tasks?

Reply:
Because the mpp client lacks some information such as ts, task id and so on.... 
Indeed, this function's name is a little perplexing, because its duty is only to arrange task to different stores  :D